diagnostic.md

```diff
@@ -54,7 +69,10 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+$('.big');
+$('#contentPane');
+$('.important.red');
+$('div > p');
 <!-- your answer ends here -->
```

Comment:
good work on selectors, just needed them for css though (no need for jquery's `$()` selectors)